CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.4.0] - 2024-09-05
```

Comment:
Please use `[Unreleased]`, the release automation will update it

```suggestion
## [Unreleased]
```

Reply:
[912f10f](https://github.com/bitmovin/bitmovin-player-react-native-analytics-conviva/pull/29/commits/912f10f218efd80288b98bb9b0a6f210b670b19d)